src/data/finset/intervals.lean

```diff
@@ -48,6 +48,12 @@ multiset.Ico.card _ _
 @[simp] theorem mem {n m l : ℕ} : l ∈ Ico n m ↔ n ≤ l ∧ l < m :=
 multiset.Ico.mem
 
+lemma coe_eq_Ico {n m : ℕ} : ↑(Ico n m) = set.Ico n m :=
```

Comment:
This should be a `simp` lemma, to make sure that most lemmas about `finset.Ico` can be deduced from the corresponding `set.Ico` lemma with `by { ext, simp [foo] }`.